api/app/Exceptions/ValidationException.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace App\Exceptions;
+
+use Illuminate\Support\MessageBag;
+use Illuminate\Http\Exception\HttpResponseException;
+
+class ValidationException extends HttpResponseException
```

Comment:
Ah, I see. I've been checking that out now. So if I understand correctly, we'll need to add handling in ```BaseController.php:renderException``` that injects the invalid object into the response. Right?